CONTRIBUTING.md

```diff
@@ -426,6 +430,7 @@ Members are listed in alphabetical order. Members are free to use the full name,
 ### Maintainers team
 
 - [Dani Guardiola @DaniGuardiola](https://github.com/DaniGuardiola)
+- [Carson McManus @dyc3](https://github.com/dyc3)
```

Comment:
Members are alphabetically listed. I think you should place yourself first before Dani

Reply:
whoops, I just woke up lol